smack-experimental/src/main/java/org/jivesoftware/smackx/muclight/element/MUCLightBlockingIQ.java

```diff
@@ -0,0 +1,125 @@
+/**
+ *
+ * Copyright 2016 Fernando Ramirez
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jivesoftware.smackx.muclight.element;
+
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+import org.jivesoftware.smack.packet.IQ;
+import org.jivesoftware.smackx.muclight.MultiUserChatLight;
+import org.jivesoftware.smackx.muclight.element.MUCLightElements.BlockingElement;
+import org.jxmpp.jid.Jid;
+
+/**
+ * MUC Light blocking IQ class.
+ * 
+ * @author Fernando Ramirez
+ *
+ */
+public class MUCLightBlockingIQ extends IQ {
+
+    public static final String ELEMENT = QUERY_ELEMENT;
+    public static final String NAMESPACE = MultiUserChatLight.NAMESPACE + MultiUserChatLight.BLOCKING;
+
+    private final boolean isSet;
+    private final boolean isGet;
```

Comment:
I removed those fields (`isSet` and `isGet`) in your MAM code. It's redundant information: The IQ class already has a field `type` which holds it. Please remove them here too.